lib/browser/api/web-contents.js

```diff
@@ -318,8 +318,14 @@ const addReturnValueToEvent = (event) => {
   })
 }
 
+let warnedAboutRendererProcessReuse = false
+
 // Add JavaScript wrappers for WebContents class.
 WebContents.prototype._init = function () {
+  if (!warnedAboutRendererProcessReuse && app._allowRendererProcessReuseIsDefaultValue) {
+    deprecate.log('The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397')
```

Comment:
want to use `deprecate.warnOnce` or similar? that pays attention to `process.noDeprecation`

Reply:
`warnOnce` logs that a thing will be removed, I can respect `process.noDeprecation` here though